generators/generator.h

```diff
@@ -32,6 +34,8 @@ namespace rpwa {
 		virtual const rpwa::particle& getGeneratedRecoil() const { return _target.recoilParticle; }
 		virtual const std::vector<rpwa::particle>& getGeneratedFinalState() const { return _decayProducts; }
 		virtual const TVector3& getGeneratedVertex() const { return _vertex; }
+		virtual const double getGeneratedXMass() const { return _xMass; }
+		virtual const double getGeneratedTPrime() const { return _tPrime; }
```

Comment:
the `const` in `virtual const double` has no effect and will lead to warnings/errors with some compilers